.github/workflows/test-warehouse.yml

```diff
@@ -82,7 +90,8 @@ jobs:
         run: sudo apt-get install python-dev libsasl2-dev gcc
 
       - name: Install dbt
-        run: pip install --pre
+        run:
+          pip install${{ (inputs.include-dbt-pre-releases == 'true' && ' --pre') || '' }}
```

Comment:
Not running on --pre by default because dbt's pre-releases are currently broken.
Added the option to enable it as a parameter, maybe we should also have a scheduled workflow that runs it (in a lower frequency)

Reply:
but from what elon told me once, we added this because we want to make sure **we** dont break with the newer versions of dbt